core/src/main/java/org/apache/accumulo/fate/AgeOffStore.java

```diff
@@ -94,6 +94,7 @@ public void ageOff() {
         try {
           switch (store.getStatus(txid)) {
             case NEW:
+            case SUBMITTED:
```

Comment:
Are we sure we want transactions in the `SUBMITTED` state to be subject to age-off? I think `NEW` makes sense because they are probably leftover from a crash and the transaction ID is just not used. However, `SUBMITTED` seems more similar to `IN_PROGRESS` here. It could still run. Right?
(Also, same question in `AgeOffStore` constructor below)

Reply:
Good catch, I was in the mode of making sure I added the enum to the switch statements. I'll update.

Reply:
Modified in f228aab